src/ducks/fagsaker.js

```diff
@@ -142,20 +142,3 @@ export const OppsummeringSelector = createSelector(
     registrertDato: saksdata.registrertDato,
   })
 );
-
-/** Hent alle arbeidsforhold og trekk ut permisjoner. I tillegg skal hver permisjon ha en kopi av arbeidsgiver
- * slik at komponenten enklere kan loope ut hver permisjon med tilhørende arbeidsgiver i en tabell.
- */
-export const PermisjonerSelector = createSelector(
```

Comment:
Ikke lenger behov for denne selectoren siden vi viser permisjoner direkte under arbeidsforhold.